src/game.cpp

```diff
@@ -7784,6 +7786,19 @@ bool pet_menu(monster *z)
         amenu.addentry(pheromone, true, 't', _("Tear out pheromone ball"));
     }
 
+    if (z->type->id == mtype_id("mon_dog") ||
+        z->type->id == mtype_id("mon_beakhound") || 
+        z->type->id == mtype_id("mon_beakhound2") || 
+        z->type->id == mtype_id("mon_beakhound3")) 
+    {
+        if (g->u.has_amount("syringe", 1)) {
+            amenu.addentry(inject, true, 'i', _("Inject mutagen"));
+        }
+        else {
```

Comment:
It would be good if your code style matched the rest of the code:
* `} else {` on one line
* Padded functions: `fun( x, y )` rather than `fun(x,y)`
* 4 spaces instead of tabs